### ISSUE TITLE
If hashtags are disabled, hashtag tab should say so

### ISSUE THREAD

**Issue by Cassolotl** [mastodon/mastodon#18226]
### Pitch

On my instance, trending hashtags in the Explore tab are disabled. The result is that nothing appears under Hashtags, but there is no explanation.

Instead I think there should be a short sentence to say "trending hashtags are disabled on this instance" or similar.

### Motivation

It would be less confusing.

**Comment by ryliejamesthomas** [comment]
Yeah, I get nothing under 'Posts' 'Hashtags' and 'News', 'cause I have all that stuff disabled. It looks a bit half-baked.

**Comment by Cassolotl** [comment]
Yeah, it looks unfinished, and if someone assumes that things won't be left unfinished then it just looks straight-up broken. :S

**Comment by progval** [comment]
My half-baked patch to hide the three columns: https://github.com/progval/mastodon/commit/082d29e137d2c27bfd6266d7851ac0277242b24f

It's not suitable for inclusion in Mastodon, but it does the job

**Comment by Cassolotl** [comment]
In 4.3, we get "Nothing is trending right now. Check back later!" Which is... an improvement? But it implies that if I check back later there might be something there, and there never is!
